.gitignore

```diff
@@ -1,6 +1,6 @@
 .DS_Store
 node_modules
-/dist
+/frontend/dist/
```

Comment:
We forgot to update the gitignore after moving `dist` to `frontend` maybe, and now the `dist` folder has been added. I'll remove it from `master` later.